src/PhpSpec/Process/Shutdown/Shutdown.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of PhpSpec, A php toolset to drive emergent
+ * design by specification.
+ *
+ * (c) Marcello Duarte <marcello.duarte@gmail.com>
+ * (c) Konstantin Kudryashov <ever.zet@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace PhpSpec\Process\Shutdown;
+
+final class Shutdown
+{
+    protected $actions;
+
+    const HHVM_FATAL_ERROR = 16777217;
+
+    public function __construct()
+    {
+        $this->actions = array();
+    }
+
+    public function registerShutdown()
+    {
+        ini_set('display_errors', '0');
+        error_reporting(E_ERROR);
```

Comment:
Is this needed? Errors aren't displayed so I would expect you can take this out.